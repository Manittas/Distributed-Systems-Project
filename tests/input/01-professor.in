list
openEnrollments 1
list
exit
