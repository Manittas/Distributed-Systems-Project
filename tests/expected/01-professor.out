> ClassState{
	capacity=0,
	openEnrollments=false,
	enrolled=[],
	discarded=[]
}

> The action completed successfully.

> ClassState{
	capacity=1,
	openEnrollments=true,
	enrolled=[],
	discarded=[]
}

> 